gensrc/script/doris_builtins_functions.py

```diff
@@ -2004,6 +2004,12 @@
         [['ipv4numtostring','inet_ntoa'], 'VARCHAR', ['SMALLINT'], 'ALWAYS_NULLABLE'],
         [['ipv4numtostring','inet_ntoa'], 'VARCHAR', ['INT'], 'ALWAYS_NULLABLE'],
         [['ipv4numtostring','inet_ntoa'], 'VARCHAR', ['BIGINT'], 'ALWAYS_NULLABLE'],
+        [['ipv4stringtonum','inet_aton'], 'BIGINT', ['VARCHAR'], 'ALWAYS_NOT_NULLABLE'],
```

Comment:
These function should be implements in Nereids(the new query planner)

Reply:
yes, I have a plan to make it the next PR

Reply:
OK, I will add.